test/functional/inspec_supermakert_test.rb

```diff
@@ -0,0 +1,27 @@
+# encoding: utf-8
+
+require 'functional/helper'
+
+describe 'inspec supermakert' do
+  include FunctionalHelper
+
+  it 'help' do
+    out = inspec('supermarket help')
+    out.exit_status.must_equal 0
+    out.stdout.must_include 'inspec supermarket exec PROFILE'
+  end
+
+  it 'info' do
+    out = inspec('supermarket info dev-sec/ssh-baseline')
+    out.exit_status.must_equal 0
+    out.stderr.must_equal ''
+    out.stdout.must_include "name: \e[0m  ssh-baseline"
+  end
+
+  it 'supermarket exec' do
+    out = inspec('supermarket exec dev-sec/ssh-baseline')
```

Comment:
Can we add a test for exit status here as well? Even if it is `wont_equal 1`.

Reply:
yeah we can do wont_equal 1, as it could be 0, 100, 101